test/hotspot/jtreg/runtime/ParallelLoad/SuperWait/SuperWaitTest.java

```diff
@@ -36,19 +36,17 @@
 
 public class SuperWaitTest {
 
-    private static Semaphore mainSync = null;
+    private static boolean dIsLoading = false;
```

Comment:
Nit: sorry just noticed the placement here. This is really an instance field of MyLoaderOne.

Reply:
I fixed it.